src/lib/select/select.ts

```diff
@@ -402,13 +402,31 @@ export class MdSelect extends _MdSelectMixinBase implements AfterContentInit, On
   }
 
   /** Event emitted when the select has been opened. */
-  @Output() onOpen: EventEmitter<void> = new EventEmitter<void>();
+  @Output() opened: EventEmitter<void> = new EventEmitter<void>();
```

Comment:
Should we make this consistent with sidenav and just have one `openedChange`?
Thoughts @kara @crisbeto?

Reply:
That's fine by me, we just have to make sure to mark it as a breaking change.